KnowSheet/scripts/check-headers.sh

```diff
@@ -10,28 +10,30 @@ set -u -e
 CPPFLAGS="-std=c++11 -g -Wall -W -DBRICKS_CHECK_HEADERS_MODE"
 LDFLAGS="-pthread"
 
-TMPDIR=.noshit
+# WARNING: TMP_DIR must be resolved from the current working directory.
```

Comment:
Perhaps it's a `NOTE`. This was a warning for me when you spotted it.

Wrt capitalization, it's like a tag. Normal sentence case does not catch the eye.

```
// Warning: Something should be here.
```
looks more like a usual comment.
```
// WARNING: Something should be here.
```
looks more like a warning.